run_docker.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+rootdir=`dirname $0`
+pushd $rootdir
+docker build -t sdk-testing -f ./scripts/docker/Dockerfile $rootdir
+docker run -v $(pwd):/opt/sdk-testing sdk-testing:latest /bin/bash -c "GO111MODULE=off && ./scripts/setup.sh && ./scripts/test.sh"
```

Comment:
I would add a `-ti` option to this so that a user can exit the test more easily if they want to

Reply:
:+1: